images/nginx/rootfs/build.sh

```diff
@@ -376,6 +376,9 @@ Include /etc/nginx/owasp-modsecurity-crs/rules/RESPONSE-999-EXCLUSION-RULES-AFTE
 # build nginx
 cd "$BUILD_PATH/openresty-$OPENRESTY_VERSION"
 
+echo "Patching NGINX for CVE-2018-16843, CVE-2018-16844, CVE-2019-9511, CVE-2019-9513, and CVE-2019-9516"
+wget -q -O - https://raw.githubusercontent.com/openresty/openresty/0a9bbae6e7927f7aad103a32badd8d81ef189a6e/patches/patch.2019.h2.txt | patch -d bundle/nginx-1.15.8/ -p0
```

Comment:
Please make sure to only use this URL as a temporary measure. The branch will disappear after the PR is merged (I will try not to force-push it before then).

Reply:
@aledbf maybe it's safer to store offline copy of the patch and apply locally?

Reply:
@thibaultcha yes, this is just to release a fix ASAP. Thanks for the heads up!

Reply:
@ElvinEfendi done